src/app/components/outcomes-info-modal/outcomes-info-modal.component.scss

```
@@ -73,8 +73,24 @@
     font-size: 2.125rem;
   }
 
+.info-modal-heading-image {
+  background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)